### PR TITLE
MFA BV100, 200, and 300 additions

### PR DESCRIPTION
**Pull request by tom-skoczylas**
Additions for primarily heat recovery, radiant panels, fan coils, makeup air, domestic hot water systems, and tanks. Please review MAU and domestic water additions very carefully. @tasodorff @tsodorff @mschulze17

### CODE REVIEW

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 1]
```diff
@@ -68,6 +68,7 @@ descriptor:
   bypass: "Route which fluid takes to bypass process."
   calendar: "A system structured around the days, weeks, and the months, used to manage schedules."
   call: "An action of speaking or meeting among multiple people."
+  carbon: "Carbon in solid form."
```
Comment (cstirdivant) [inline ontology/yaml/resources/subfields/subfields.yaml:71]: Not necessarily solid form.

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 2]
```diff
@@ -1079,3 +1079,41 @@ literals:
...
+- heat_recovery_run_command:
+  - ON
+  - OFF
+- exhaust_air_differential_pressure_sensor
+- leaving_heat_recovery_coil_temperature_sensor
+- zone_floor_temperature_sensor
+- zone_floor_temperature_setpoint
+- entering_heat_recovery_coil_temperature_sensor
+- carbon_filter_differential_pressure_sensor
+- heating_supply_water_isolation_valve_status
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:1107]: add states

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 3]
```diff
@@ -4399,4 +4402,129 @@ HWDDC:
...
+  - outside_air_flowrate_sensor_2
+  uses:
+  - outside_air_temperature_sensor
+  - outside_air_damper_percentage_command_1
+  - outside_air_damper_percentage_command_2
+  implements:
+  - MONITORING
+ 
+EHHRC:
+  description: "Exhaust hydronic heat recovery coil on exhaust to be used elsewhere."
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4435]: revise description

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 4]
```diff
@@ -4399,4 +4402,129 @@ HWDDC:
...
+  is_abstract: true
+  implements:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4437]: please put implements field at the end, see other type structures for precedent

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 5]
```diff
@@ -4399,4 +4402,129 @@ HWDDC:
...
+  - leaving_heat_recovery_coil_temperature_sensor
+  - entering_heat_recovery_coil_temperature_sensor
+ 
+DPBHCC:
+  description: "Two-pipe binary (open/closed) heating and cooling control. Has a single control valve
+  that is fed by two separate headers for heating and cooling water. There is an isolation valve for
+  each incoming system and a single control valve. Valve and mode control to zone temperature 
+  (heating/cooling setpoint configuration)."
+  is_abstract: true
+  implements:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4452]: move section to end of type

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 6]
```diff
@@ -4399,4 +4402,129 @@ HWDDC:
...
+  - heating_supply_water_isolation_valve_command
+  - heating_supply_water_isolation_valve_status
+  - zone_air_cooling_temperature_setpoint
+  - zone_air_heating_temperature_setpoint
+  - zone_air_temperature_sensor
+ 
+FTC:
+  description: "Floor temperature control"
+  is_abstract: true
+  implements:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4470]: move to end of type

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 7]
```diff
@@ -4399,4 +4402,129 @@ HWDDC:
...
+  - MONITORING
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4471]: this implements OPERATIONAL

File: ontology/yaml/resources/HVAC/entity_types/DHWT.yaml [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+########################
+
+DHWT_HWTTC:
+  description: "Domestic hot water tank."
+  is_canonical: true
+  implements:
+  - DHWT
+  - HWTTC
+
+DHWT_PHTTC:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/DHWT.yaml:26]: typo

**Review by cstirdivant — CHANGES_REQUESTED** [review]

**Review by cstirdivant — APPROVED** [review]

**Comment by charbull** [comment]
@tasodorff @cstirdivant can this PR be merged?

**Comment by cstirdivant** [comment]
@charbull please wait to merge, Trevor & I are reviewing tomorrow.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 9]
```diff
@@ -4409,7 +4417,130 @@ HWDDC:
   implements:
   - CONTROL
 
+# New abstracts for MFA BV100 HVAC
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4420]: Remove references to specific sites, if possible.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 10]
```diff
@@ -4409,7 +4417,130 @@ HWDDC:
...
+CFDPM:
+  description: "Carbon filter pressure monitoring."
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4422]: Please add to description, something like: "Carbon filter pressure monitoring, where specific filter type is required."

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 11]
```diff
@@ -4409,7 +4417,130 @@ HWDDC:
...
+  is_abstract: true
+  uses:
+  - carbon_filter_differential_pressure_sensor
+  implements:
+  - MONITORING
+ 
+VOADM2X:
+  description: "Variable outside air damper monitoring."
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4430]: add to description: "[...] where there are two separate, equal sets of dampers that operate in conjunction."

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 12]
```diff
@@ -4409,7 +4417,130 @@ HWDDC:
...
+  - leaving_heat_recovery_coil_temperature_sensor
+  - entering_heat_recovery_coil_temperature_sensor
+  - exhaust_air_flowrate_sensor
+  implements:
+  - MONITORING
+ 
+DPBHCC:
+  description: "Two-pipe binary (open/closed) heating and cooling control. Has a single control valve
+  that is fed by two separate headers for heating and cooling water. There is an isolation valve for
+  each incoming system and a single control valve. Valve and mode control to zone temperature
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4461]: you mention a single control valve, but I dont see it defined on the type.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 13]
```diff
@@ -4409,7 +4417,130 @@ HWDDC:
...
+  - heating_supply_water_isolation_valve_command
+  - heating_supply_water_isolation_valve_status
+  - zone_air_cooling_temperature_setpoint
+  - zone_air_heating_temperature_setpoint
+  - zone_air_temperature_sensor
+  implements:
+  - CONTROL
+ 
+FTC:
+  description: "Floor temperature control"
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4480]: please add a bit more here: "[...] where the temperatures are embedded in the floor (as opposed to open to the air)."

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 14]
```diff
@@ -4409,7 +4417,130 @@ HWDDC:
...
+  - supply_air_temperature_sensor
+  - supply_air_temperature_setpoint
+  - supply_water_valve_percentage_command
+  - heat_recovery_run_command
+  - heat_recovery_supply_water_isolation_valve_command
+  - chilled_supply_water_isolation_valve_command
+  implements:
+  - CONTROL
+
+# New abstracts for MFA BV100 DHW
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4512]: same, remove bldg ref

**Review by tasodorff — CHANGES_REQUESTED** [review]
Please make some minor additions to descriptions and remove building references.

**Review by cstirdivant — APPROVED** [review]

**Review by tasodorff — APPROVED** [review]

**Pull request merged**